scripts/update-ios-examples.mjs

```diff
@@ -9,12 +9,14 @@ import fs from "fs/promises";
 const documentationPaths = [
   "platform/ios/MapLibre.docc/GettingStarted.md",
   "platform/ios/MapLibre.docc/LineOnUserTap.md",
+  "platform/ios/MapLibre.docc/LocationPrivacyExample.md",
```

Comment:
Can't you use glob or something that doesn't require adding a page every time?

Reply:
My idea was not parsing files unnecessarily, since some of them will not contain examples or do not need examples replaced, but since there are not that many files it's fine to go through them all.